src/sagas/wallet.js

```diff
@@ -531,10 +546,47 @@ export function* onWalletConnStateUpdate({ payload }) {
   yield put(isOnlineUpdate({ isOnline }));
 }
 
+export function* walletReloading() {
+  console.log('Wallet reloading.');
```

Comment:
Remove

Reply:
Done in 2a6522688c8969fbd09a2c81ce76093d51ac53d1